frappe/public/js/frappe/views/reports/report_view.js

```diff
@@ -896,7 +896,9 @@ frappe.views.ReportView = class ReportView extends frappe.views.ListView {
 			.filter(standard_fields_filter);
 
 		// filter out docstatus field from picker
-		let std_fields = frappe.model.std_fields.filter((df) => df.fieldname !== "docstatus");
+		let std_fields = frappe.model.std_fields.filter(
+			(df) => df.fieldname !== "docstatus" && df.fieldname[0] !== "_"
```

Comment:
```suggestion
			(df) => !["docstatus", "_comments"].includes(df.fieldname)
```


Other fields seem to be human readble. 
- _user_tags - comma separated tags
- _assign - list of assigned users
- _liked_by - list of users who liked